### ISSUE TITLE
Locale selector: Select region button which triggers the modal dialog

### ISSUE THREAD

**Issue by chetankan** [carbon-design-system/ibm-dotcom-library#1765]
Issue:- Screen reader announces the currently selected option instead of the label.

Expected:- Screen reader should announce the label i.e. "select your region"

URL:- http://ibmdotcom-nextjs-test.mybluemix.net/learn.html

**Comment by RaphaelAmadeu** [comment]
@chetankan 
Could you please test this again?
Thank you!
